### PR TITLE
rhn_register: add support for profilename

### PR DESCRIPTION
**Pull request by gerhardsam**

### CODE REVIEW

**Comment by gregdek** [comment]
Hey, @jlaska, looks like this module is one of yours!  Could you please give this a quick review?  If you give it a +1, we'll tag P1_reviewed for faster inclusion.

**Comment by jlaska** [comment]
@gregdek Unfortunately, I'm not able to test at the moment.  However, the changes look good to me.

@gerhardsam Thanks for the contribution!

**Comment by gregdek** [comment]
Our inability to test this makes it less safe to merge until we can remedy that.  I'll take this issue for now.

**Comment by gregdek** [comment]
Per @jlaska, it's safe enough. Promoting to P1_reviewed.

**Comment by gerhardsam** [comment]
@gregdek Thanks for reviewing!

File: packaging/os/rhn_register.py [hunk 1]
```diff
@@ -56,6 +56,11 @@
             - supply an activation key for use with registration
         required: False
         default: null
+    profilename:
```
Comment (bcoca) [inline packaging/os/rhn_register.py:59]: needs version_added for this feature

**Comment by gerhardsam** [comment]
@bcoca Just for clarification: Has that to be added by myself ore one of the reviewers?

**Comment by abadger** [comment]
@gerhardsam For the future: In general the author of the code does that.  But it's a simple change and I can do it after I merge :-)

**Comment by abadger** [comment]
version_added added in 4ef5a45

@gerhardsam Thanks for the PR!

# Merged

Hi!

This has been merged in, and will also be included in the next major release.

If you or anyone else has any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!

**Pull request merged**